README.md

```diff
@@ -41,6 +41,10 @@
 - Install `bubblesub`: `pip install --user .`
 - Run bubblesub: `python3 -m bubblesub` or simply `bubblesub`
 
+If you want to simplify `bubblesub` installation, look at our [Dockerfile](Dockerfile)!
```

Comment:
```diff
-!
+.
```

Reply:
I liked the emphasis given by the exclamation mark, but done